Base/Python/slicer/tests/test_slicer_util_load_data.py

```diff
@@ -0,0 +1,36 @@
+import tempfile
+import unittest
+from pathlib import Path
+from urllib.parse import urljoin
+
+import slicer
+
+
+class SlicerUtilLoadDataTests(unittest.TestCase):
+
+    def setUp(self):
+
+        emm = slicer.app.extensionsManagerModel()
+        is_ext_installed = emm.isExtensionInstalled("SlicerDMRI")
+        self.assertTrue(is_ext_installed)
```

Comment:
```suggestion
```

The test should always pass independently of having the extension installed or not

Reply:
Removed in bc1d677.